src/parseTools.js

```diff
@@ -442,24 +442,25 @@ function makeGetValue(ptr, pos, type, noNeedFirst, unsigned, ignore, align, noSa
  *             'null' means, in the context of SAFE_HEAP, that we should accept all types;
  *             which means we should write to all slabs, ignore type differences if any on reads, etc.
  * @param {bool} noNeedFirst Whether to ignore the offset in the pointer itself.
+ * @param {bool} unsigned Indicating signed-ness
  * @param {bool} ignore: legacy, ignored.
  * @param {number} align: TODO
  * @param {bool} noSafe: TODO
  * @param {string} sep: TODO
  * @param {bool} forcedAlign: legacy, ignored.
  * @return {TODO}
  */
-function makeSetValue(ptr, pos, value, type, noNeedFirst, ignore, align, noSafe, sep = ';', forcedAlign) {
+function makeSetValue(ptr, pos, value, type, noNeedFirst, unsigned, ignore, align, noSafe, sep = ';', forcedAlign) {
```

Comment:
Add a new argument in this position will bread existing callers.

I'd rather not add complexity to makeSetValue by adding another argument here.

Reply:
(also I'm curious why this was never needed before... how is it that all of the `makeSetValue` calls works just fine without ever using the unsigned heaps..).